lib/src/store.rs

```diff
@@ -118,6 +127,75 @@ impl Store {
         self.get_commit_async(id).block_on()
     }
 
+    pub fn is_local_change(self: &Arc<Self>, id: &ChangeId) -> BackendResult<bool> {
```

Comment:
Implemented as a lazy-loaded json file, with a write-through cache. Given that it will not be modified very often writing the whole set every time seems fine. Happy to change any technical details.